docs/superpowers/specs/2026-05-21-android-import-service-account-design.md

```
@@ -0,0 +1,332 @@
+  `client.commit_edit(package_name, edit_id, …)` etc.
+- `supply/lib/supply/options.rb` exposes `package_name`, `json_key`,
+  `json_key_data`, `issuer`, etc. — **no** `developer_id` /
+  `developer_account_id` option.
+
+So for imports the developer ID would only be useful as a validation aid, and
+even then `edits.insert(packageName)` is a strictly better check (it tests the
+same auth path fastlane will take at build time).
+
+## Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)